wgsl/index.bs

```diff
@@ -1785,26 +1868,16 @@ variable_decl
   : VAR variable_storage_decoration? variable_ident_decl
 
 variable_ident_decl
-  : IDENT COLON decoration_list* type_decl
+  : IDENT COLON attribute_list* type_decl
 
 variable_storage_decoration
   : LESS_THAN storage_class GREATER_THAN
 
 </pre>
 
-<table class='data'>
-  <thead>
-    <tr><th>Variable declaration decoration keys<th>Valid values<th>Note
-  </thead>
-  <tr><td>`access`<td>`read`, `write` or `read_write`<td>
-</table>
-
-The access decoration must only appear on a type used as the store type for a
-variable in the [=storage classes/storage=] storage class.
-The access decoration must not appear
-on a type of const declaration nor as the store type for variable with a
-storage class other than [=storage classes/storage=]. The access decoration is required for
-variables in the [=storage classes/storage=] storage class.
+Variables in the [=storage classes/storage=] storage class and variables with a
+(storage texture)[#storage-textures] type must have an [=access=] attribute
```

Comment:
typo on the link. It renders as:


variables with a (storage texture)[#storage-textures] type